integration-test-processor/src/main/java/org/springframework/cloud/stream/module/it/IntegrationTestProcessorProperties.java

```diff
@@ -56,6 +57,12 @@
 	 */
 	private String parameterThatMayNeedEscaping;
 
+	/*
+	 * Same as ChannelBindingServiceProperties.instanceIndex.

```

Comment:
it doesn't matter if it's the same as `ChannelBindingServiceProperties.instanceIndex` - that's inverting the relationship... everything about this app should be as general as possible - anything related to `spring-cloud-stream` apps is more specific than this